examples/object_detection/model/coco_labels_list.txt

```diff
@@ -0,0 +1,91 @@
+???
+person
+bicycle
+car
+motorcycle
+airplane
+bus
+train
+truck
+boat
+traffic light
+fire hydrant
+???
+stop sign
+parking meter
+bench
+bird
+cat
+dog
+horse
+sheep
+cow
+elephant
+bear
+zebra
+giraffe
+???
+backpack
+umbrella
+???
+???
```

Comment:
I am not sure if so many ??? here are correct, but is is the same as the upstream file https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/examples/android/app/src/main/assets/coco_labels_list.txt

Reply:
Please look at here:
https://stackoverflow.com/questions/50665110/tensorflow-models-uses-coco-90-class-ids-although-coco-has-only-80-categories

I guess the dataset for this object detection model is the 2014 released version(only contains 80 object categories).